src/pe_reports/data/data_schema.sql

```diff
@@ -558,6 +558,35 @@ CREATE TABLE public.unique_software (
     software_name text NOT NULL
 );
 
+--
+--Name: was_tracker_customerdata
+--
+
+create table was_tracker_customerdata
+(
+    customer_id              uuid default uuid_generate_v1() not null
+        constraint was_tracker_customerdata_pk
+            primary key,
+    tag                      text                            not null,
+    customer_name            text                            not null,
+    testing_sector           text                            not null,
+    ci_type                  text                            not null,
+    ticket                   text                            not null,
+    next_scheduled           text                            not null,
+    last_scanned             text                            not null,
+    frequency                text                            not null,
+    comments_notes           text                            not null,
+    was_report_poc           text                            not null,
+    was_report_email         text                            not null,
+    onboarding_date          text                            not null,
+    no_of_web_apps           integer                         not null,
+    no_web_apps_last_updated text,
+    elections                text
```

Comment:
Can these fields be alphabetized or does their order matter here?  Alphabetical ordering of long lists like this is always preferable for readability, as long as the order does not matter.

Reply:
@dav3r , 
The WAS data isn't really mixing with our data at any level at this point and no real talk of integration with their data. We are just providing a space for their data and API endpoints for their team members interact with their data. I left the columns in the order that they had them so I dont change their process of writing to their flat files as they have code in place to write the flat file in this format. I do agree with you the columns s/b in alphabetical order, just doesn't fit the current process.

Reply:
If you are just giving them a place to put their data and there are no plans to integrate it with P&E data, is there any reason why they are not getting their own database, as opposed to adding a (essentially unrelated) WAS table in the P&E database?

Reply:
I should rephrase that, our data isn't twisted with theirs, but we will be pulling the data to report on the data in our 
monthly reports.

Reply:
Got it- so it sounds like there is a need to have WAS data in the P&E database.  Thanks for clarifying!